web/pgadmin/browser/server_groups/servers/static/js/variable.ui.js

```diff
@@ -67,24 +73,64 @@ export default class VariableSchema extends BaseUISchema {
     });
   }
 
+  getPlaceHolderMsg(variable) {
+    let msg = '';
+    if (variable?.min_server_version && variable?.max_server_version) {
+      msg = gettext(variable?.max_server_version + ' <= Supported version >= ' + variable?.min_server_version );
```

Comment:
Please use the syntax `gettext('%s <= Supported version >= %s', var1, var2)`

Reply:
Done